mongo_repository.go

```diff
@@ -288,6 +288,38 @@ func (r *MongoRepository[T]) lastSnapshot(ctx context.Context, objectID string)
 	return &lastSnapshot, nil
 }
 
+func (r *MongoRepository[T]) Remove(ctx context.Context, objectID string, object T) (T, error) {
```

Comment:
```suggestion
func (r *MongoRepository[T]) Remove(ctx context.Context, objectID string, object T) error {
```

Suite au commentaire du inmemory repository sur le load. Je pense qu'il n'est pas utile de renvoyer l'objet supprimé ici ni de le load